packages/Operators/test/reference_implementation/DTK_ReferenceHexMesh.hpp

```diff
@@ -66,7 +66,21 @@ class ReferenceHexMesh
   public:
 
     /*!
-     * \brief Constructor.
+     * \brief Num cells constructor.
+     */
+    ReferenceHexMesh( const Teuchos::RCP<const Teuchos::Comm<int> >& comm,
+                      double x_min,
+                      double x_max,
+                      const int x_num_cells,
+                      double y_min,
+                      double y_max,
+                      const int y_num_cells,
+                      double z_min,
+                      double z_max,
+                      const int z_num_cells );
```

Comment:
Oh. That works but it's a bit cumbersome. I thought more a signature like:
```
ReferenceHexMesh( const Teuchos::RCP<const Teuchos::Comm<int> >& comm,
    std::vector<double> const &p1,
    std::vector<double> const &p2,
    std::vector<int> const &subdivisions );
```
so you can call `ReferenceHexMesh mesh(comm, {0.0, 0.0, 0.0}, {2.0, 2.0, 2.0}, {8, 8, 8});`
I like the good ole standard vectors because of the list initialization but `Theuchos::array<T>` works fine here too.